src/server.js

```diff
@@ -146,16 +156,11 @@ function index(req, res, next) {
 		pkg,
 		Helper.config
 	);
+	if (!data.theme.includes(".css")) { // Backwards compatibility for old way of specifying themes in settings
```

Comment:
Mind adding a deprecation notice like the following in this block?

```js
log.warn(`Referring to CSS files in the ${colors.green("theme")} setting of ${colors.green(Helper.CONFIG_PATH)} is ${colors.bold("deprecated")} and will be removed in a future version.`);
```

Unless of course we are not planning to remove support for CSS files, but I think we will eventually, no?

Reply:
Yeah, sure.

Reply:
It's actually not inside this block as we go into this block if we are using the new way of doing it, so it's in an else block.